### ISSUE TITLE
Handson Suggestion/Feedbacks

### ISSUE THREAD

**Issue by vipinvkmenon** [SAP-samples/teched2023-XP263#2]
Here are some suggestions based on our observation:

- The Requirements for Azure Scenario seem to be missing or unclear.
- Exercise 1.3.5 -> Images missing...only Alt Text is seen

These are merely suggestions and not set in stone. As the contributors for the repo you are always free to implement in the way you feel is best :). You may close the issue accordingly.
